### ISSUE TITLE
Arrow Key does not select Radio Button when Restoring Previous Version of FileSet

### ISSUE THREAD

**Issue by chrisdaaz** [samvera/hyrax#3452]
### Descriptive summary

Hyrax Master 3.0.0 beta

### Rationale

Keyboard users may need to restore previous versions of filesets.

### Expected behavior

When selecting a version to restore a FileSet, keyboard users should be able to use an arrow key to select the option before executing that selection.

### Actual behavior

The radio button cannot be selected via keyboard:

![image](https://user-images.githubusercontent.com/24395592/49244287-0dce2f00-f3d5-11e8-82ea-af7d8c08bcb2.png)

### Steps to reproduce the behavior

1. Go to the Edit Versions page for a FileSet
2. Add a new version to the file
3. Save the new version
4. Attempt to restore to the previous version of the fileset using your keyboard.

### Related work

Link to related tickets or prior related work here.

**Comment by chrisdaaz** [comment]
nevermind, didn't know that i needed to press the space bar.
